packages/internal/src/lib-chainStorage.js

```diff
@@ -34,6 +35,15 @@ const { Fail } = assert;
  * @property {(subPath: string, options?: {sequence?: boolean}) => StorageNode} makeChildNode
  */
 
+const ChainStorageNodeI = M.interface('StorageNode', {
+  setValue: M.callWhen(M.string()).returns(),
+  getPath: M.call().returns(M.string()),
+  getStoreKey: M.callWhen().returns(M.record()),
+  makeChildNode: M.call(M.string())
+    .optional(M.record())
+    .returns(M.remotable('StorageNode')),
```

Comment:
Should this be more specific?
```suggestion
  makeChildNode: M.call(M.string())
    .optional(M.splitRecord({}, { sequence: M.boolean() }))
    .returns(M.remotable('StorageNode')),
```
or even reject unknown options?
```suggestion
  makeChildNode: M.call(M.string())
    .optional(M.splitRecord({}, { sequence: M.boolean() }, {}))
    .returns(M.remotable('StorageNode')),
```